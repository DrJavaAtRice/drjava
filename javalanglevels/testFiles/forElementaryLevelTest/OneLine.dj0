class Maggie extends Object { int i, j;
  int foo() {
    System.out.println("Hello World.");
    return i;
  }
}
class OneLine extends Maggie { 
  int stuff; 
  double method(char c) { return stuff; } 
}