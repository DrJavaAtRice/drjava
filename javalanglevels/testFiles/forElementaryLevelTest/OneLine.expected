// Language Level Converter line number map: dj*->java. Entries: 10
//     1->4         2->5         3->6         4->7         5->8         6->9         7->57        8->58   
//     9->59       10->60   
class Maggie extends Object { private final int i, j;
  public int foo() {
    System.out.println("Hello World.");
    return i;
  }

  /** This method is automatically generated by the Language Level Converter. */
  public Maggie(int i, int j) {
    super();
    this.i = i;
    this.j = j;
  }

  /** This method is automatically generated by the Language Level Converter. */
  public int i() {
    return i;
  }

  /** This method is automatically generated by the Language Level Converter. */
  public int j() {
    return j;
  }

  /** This method is automatically generated by the Language Level Converter. */
  public java.lang.String toString() {
    return getClass().getName() + "(" + 
        i() + ", " + 
        j() + 
        ")";
  }

  /** This method is automatically generated by the Language Level Converter. */
  public boolean equals(java.lang.Object o) {
    if (this == o) {
      return true;
    }
    else if ((o == null) || (! o.getClass().equals(getClass()))) {
      return false;
    }
    else {
      Maggie cast = ((Maggie) o);
      return (i() == cast.i()) && 
          (j() == cast.j());
    }
  }

  /** This method is automatically generated by the Language Level Converter. */
  public int hashCode() {
    return getClass().hashCode() ^ 
        i() ^ 
        j();
  }
}
class OneLine extends Maggie { 
  private final int stuff; 
  public double method(final char c) { return stuff; } 

  /** This method is automatically generated by the Language Level Converter. */
  public OneLine(int super_i, int super_j, int stuff) {
    super(super_i, super_j);
    this.stuff = stuff;
  }

  /** This method is automatically generated by the Language Level Converter. */
  public int stuff() {
    return stuff;
  }

  /** This method is automatically generated by the Language Level Converter. */
  public java.lang.String toString() {
    return getClass().getName() + "(" + 
        stuff() + ", " + 
        i() + ", " + 
        j() + 
        ")";
  }

  /** This method is automatically generated by the Language Level Converter. */
  public boolean equals(java.lang.Object o) {
    if (this == o) {
      return true;
    }
    else if ((o == null) || (! o.getClass().equals(getClass()))) {
      return false;
    }
    else {
      OneLine cast = ((OneLine) o);
      return (stuff() == cast.stuff()) && 
          (i() == cast.i()) && 
          (j() == cast.j());
    }
  }

  /** This method is automatically generated by the Language Level Converter. */
  public int hashCode() {
    return getClass().hashCode() ^ 
        stuff() ^ 
        i() ^ 
        j();
  }
}
